### ISSUE TITLE
Self Service's custom icon not displaying as `overlayicon` when SYM is executed via an `enrollmentComplete` trigger

### ISSUE THREAD

**Issue by dan-snelson** [dan-snelson/dialog-scripts#40]
See [swiftDialog Issue No. 235](https://github.com/bartreardon/swiftDialog/issues/235).

**Describe the bug**
- When Setup Your Mac is executed via an `enrollmentComplete` trigger, Self Service's icon is not displaying as expected.
- See [this thread](https://macadmins.slack.com/archives/C04FRRN3281/p1676667546693919) and
- [This thread](https://macadmins.slack.com/archives/C04FRRN3281/p1677086968968949)

**To Reproduce**
 - Execute SYM via an `enrollmentComplete` trigger
 
**Expected behavior**
- Self Service's custom icon displays as `overlayicon`

**Code/log output**
```
su - "${loggedInUser}" -c "/usr/bin/open -a \"${overlayicon}\" -g -j"
```

**Comment by dan-snelson** [comment]
Thanks, [@GadgetGeekNI](https://macadmins.slack.com/archives/C04FRRN3281/p1677077145110089?thread_ts=1676667546.693919&cid=C04FRRN3281) (and @mactroll)

```
# If SELF_SERVICE_CUSTOM_BRANDING is set to true. Loading the updated icon
  if [ "$SELF_SERVICE_CUSTOM_BRANDING" = true ]; then
    open -a "/Applications/$SELF_SERVICE_APP_NAME" --hide

    # Loop waiting on the branding image to properly show in the users library - wait up to 20 seconds
     SELF_SERVICE_COUNTER=0
     CUSTOM_BRANDING_PNG="/Users/$CURRENT_USER/Library/Application Support/com.jamfsoftware.selfservice.mac/Documents/Images/brandingimage.png"

     until [ -f "$CUSTOM_BRANDING_PNG" ]; do
       echo "$(date "+%a %h %d %H:%M:%S"): Waiting for branding image from Jamf Pro." >> "$DEP_NOTIFY_DEBUG"
        sleep 1
        (( SELF_SERVICE_COUNTER++ ))
        if [ $SELF_SERVICE_COUNTER -gt $SELF_SERVICE_CUSTOM_WAIT ];then
            CUSTOM_BRANDING_PNG="/Applications/Self Service.app/Contents/Resources/AppIcon.icns"
            break
        fi
     done
```

**Comment by dan-snelson** [comment]
Addressed in [1.7.2-rc2](https://github.com/dan-snelson/dialog-scripts/commit/7bdce45faa8ec963557b1ec4057c77b90958ebea).

**Comment by dan-snelson** [comment]
Addressed in [1.7.2](https://snelson.us/2023/02/setup-your-mac-via-swiftdialog-1-7-0/#1-7-2-28-feb-2023).
